### ISSUE TITLE
`use_hash_url` support

### ISSUE THREAD

**Issue by rambip** [Synphonyte/leptos-use#38]
Leptos natively support `create_query_signal` in leptos-router, but in some cases you would prefer using the hash-url instead.

Like if I share a link to the leptos docs, there will be a hash symbol in it refering to the chapter (https://leptos-rs.github.io/leptos/router/18_params_and_queries.html?highlight=use_que#params-and-queries)

Maybe I should open this issue in `leptos-router` instead, I don't know.

**Comment by maccesch** [comment]
Yeah I think this should be in the router. If they refuse you can re-open here.
